tests/templates/kuttl/logging/trino-vector-aggregator-values.yaml.j2

```diff
@@ -104,13 +104,23 @@ customConfig:
       condition: >-
         .pod == "test-trino-worker-custom-log-config-0" &&
         .container == "prepare"
+    filteredInvalidEvents:
+      type: filter
+      inputs: [vector]
+      condition: |-
+        .timestamp == from_unix_timestamp!(0) ||
+        is_null(.level) ||
+        is_null(.logger) ||
+        is_null(.message)
   sinks:
     out:
-      inputs: [automaticLogConfig*, customLogConfig*]
+      inputs: [filtered*, custom*]
```

Comment:
Why didn't you also prefix the custom ones with "filtered"?

Reply:
To preserve the previous logical separation between `custom` and `automatic`. I didn't look at the details of the `custom` entries. Should I do it ?

Reply:
There is no need to look at the details. Just prefix all rules with `filtered` (see e.g. https://github.com/stackabletech/druid-operator/pull/459/files#diff-d35089690058957988f8dd6b3bcf295ab343dd5fe8db9381dbd1daeb3c537728).

Reply:
Done https://github.com/stackabletech/trino-operator/pull/453/commits/fcdae35357cab1995e108259dea47f72cc754c4c